sympy/physics/mechanics/rigidbody.py

```diff
@@ -182,7 +182,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H of a rigid body B about some point O in a frame
         N is given by:
 
-            H = I . w + r x Mv
+            `H = dot(I, w) + cross(r, M*v)`
```

Comment:
Could you make the indentations of this equation equal with respect to the other docstring equations? What I mean is the consistent usage of one of the following in each of these docstrings:
```py
        `equation`
```
or
```py
            `equation`
```